test-common/test-common-runtime/src/main/java/org/apache/kafka/common/test/RaftClusterInvocationContext.java

```diff
@@ -14,18 +14,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.common.test.junit;
+package org.apache.kafka.common.test;
 
 import kafka.network.SocketServer;
 import kafka.server.BrokerServer;
 import kafka.server.ControllerServer;
 import kafka.server.KafkaBroker;
 
 import org.apache.kafka.common.network.ListenerName;
-import org.apache.kafka.common.test.ClusterInstance;
-import org.apache.kafka.common.test.KafkaClusterTestKit;
-import org.apache.kafka.common.test.TestKitNodes;
-import org.apache.kafka.common.test.TestUtils;
```

Comment:
It seems this class uses only one method of `TestUtils`. Maybe we can just eliminate the reference without changing the package?